ExtensionTemplates/ExtensionInterfaceTemplate.txt

```diff
@@ -3,8 +3,8 @@ using Microsoft.MixedReality.Toolkit.Extensions;
 
 namespace #NAMESPACE#
 {
-   public interface #INTERFACE_NAME# : IMixedRealityExtensionService
-    {
-        // Expose service features and abilities here
-    }
+	public interface #INTERFACE_NAME# : IMixedRealityExtensionService
```

Comment:
Do these need to be tabs? Or can they be spaces, like the code formatting?

Reply:
I was getting some funky results with spaces. Not sure why.

Reply:
Any chance we could do a tabs -> spaces conversion in CreateTextAssetFromTemplate to fix that part up?

Reply:
Or I guess we just depend on peoples' text editors to convert the automagically.